Library/Homebrew/cask/dsl/caveats.rb

```diff
@@ -114,15 +114,16 @@ def eval_caveats(&block)
             #{@cask} requires Java. You can install the latest version with:
               brew install --cask adoptopenjdk
           EOS
-        elsif java_version.include?("11") || java_version.include?("+")
+        elsif java_version.include?("+")
```

Comment:
I wonder why `11` was specified here already and if there's another value/values that should be here.

Reply:
Since the [inception of the file](https://github.com/Homebrew/brew/blame/687f0fcf721c8e36f32570ed72d0988a6eaf986f/Library/Homebrew/cask/lib/hbc/dsl/caveats.rb#L49), this value always tracked the current stable release of Java - advancing to [9](https://github.com/Homebrew/brew/commit/208135f59f7a02bd9329c0995a4aa74166c0b2d0), [10](https://github.com/Homebrew/brew/commit/7603d05d053db11c7a7606d5fc414cee3b23df13), and finally [11](https://github.com/Homebrew/brew/commit/d3fea843878758eb65c810d8300437bd554d0888). While Java continued to release major versions since then, Homebrew didn't track these changes. One option would be to resume tracking it - but in my opinion, any value with a `+` quantifier implies "current stable release", so there's really no benefit to tracking the current version. The real purpose of `+` is to establish a minimum compatible version and to encourage users to install the current version.

Reply:
Would it be worth parsing this from the Java cask if it's available?

Reply:
Yes, I think that would be a nice approach (although implementing such parsing of the Java cask is beyond my abilities, I'm afraid).